WEEK-3 FUNCTIONS/08_ebubekir/polindrom_words.py

```diff
@@ -0,0 +1,11 @@
+"""
+Write a function that controls the given inputs 
+whether they are equal to their reversed order or not.
+Example:
+Input >>> madam, tacocat, utrecht
+Output >>> True, True, False"""
+
```

Comment:
args=["madam","level","father","pop","tacocat","utrect"]#define list
def pal(args):
    palindrome=list(map(lambda string: string == string[::-1],args))    #if the string =revrsed(string), take it in the list
    print(palindrome)

pal(args)

Fonksiyon seklinde yapmaniz gerekiyordu.